pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/plugin/internal/ValidationPlugin.java

```diff
@@ -112,7 +118,7 @@ public void apply(final Project project) {
         /*
          * Run mypy.
          *
-         * This uses the mypy.ini file if present to configure mypy.
+         * This uses the mypy.ini or setup.cfg file if present to configure mypy.
```

Comment:
Perhaps mention `setup.cfg` first?  I'd like to deprecate use of `mypy.ini`.  One cfg to rule them all! :)

Reply:
Good point! Changed it.. Although, somewhat related, it seems that `black` is only supporting pyproject.toml, so it seems the fragmentation continues :(